### DISCUSSION TITLE
How to disable SSL?

### DISCUSSION THREAD

**Discussion by kaihendry** [php/frankenphp#1078] [Q&A]
How do I disable SSL for testing purposes? Out of the box it appears to redirect to SSL!

```
curl -i http://localhost/
HTTP/1.1 308 Permanent Redirect
Connection: close
Location: https://localhost/
Server: Caddy
Date: Wed, 09 Oct 2024 12:25:41 GMT
Content-Length: 0
```

**Comment by kitro** [comment]
You can set auto_https to `off` or `disable_redirects`
https://caddyserver.com/docs/caddyfile/options#auto-https

Also you can listen to 80 port only
```
{{ DOMAINE_NAME }}:80 { 
....
}
```

**Comment by kaihendry** [comment]
I ended up doing something like:
```
{
	frankenphp
}

web:80, docker:80, localhost:80 {
	root * public/
	encode zstd br gzip
	php_server
}
```

The [minimal example](https://frankenphp.dev/docs/config/#caddyfile-config) critically omitted the root bit!
